src/coreclr/vm/jitinterface.cpp

```diff
@@ -1570,9 +1580,7 @@ void CEEInfo::getFieldInfo (CORINFO_RESOLVED_TOKEN * pResolvedToken,
                 fieldAccessor = CORINFO_FIELD_STATIC_SHARED_STATIC_HELPER;
 
                 pResult->helper = getSharedStaticsHelper(pField, pFieldMT);
-
-#ifdef HOST_WINDOWS
-#ifndef TARGET_ARM
+#if defined(TARGET_ARM) || (!defined(HOST_WINDOWS) && defined(TARGET_32BIT))
```

Comment:
Is this ifdef right? We are explicitly enabling this for `TARGET_ARM` while we used to disable it.

Also, the "For windows" part of the comment below is out of sync with this condition.

Reply:
ah yes. Fixed.